CMakeLists.txt

```diff
@@ -385,8 +384,23 @@ message(STATUS "summary of build options:
       GnuTLS:         ${HAVE_GNUTLS} (LIBS='${GNUTLS_LIBRARIES}')
       BoringSSL:      ${HAVE_BORINGSSL} (LIBS='${BORINGSSL_LIBRARIES}')
       Picotls:        ${HAVE_PICOTLS} (LIBS='${PICOTLS_LIBRARIES}')
-      wolfSSL:        ${HAVE_WOLFSSL} (LIBS='${WOLFSSL_LIBRARIES}')
+      wolfSSL:        ${HAVE_WOLFSSL}
       Jemalloc:       ${HAVE_JEMALLOC} (LIBS='${JEMALLOC_LIBRARIES}')
       Libbrotlienc:   ${HAVE_LIBBROTLIENC} (LIBS='${LIBBROTLIENC_LIBRARIES}')
       Libbrotlidec:   ${HAVE_LIBBROTLIDEC} (LIBS='${LIBBROTLIDEC_LIBRARIES}')
 ")
+
+set(_libs_msg)
```

Comment:
this snippet attempts to reproduce previous summary of all imported libs. Example output:

```
-- Libs:
    wolfssl[RELEASE]: /Users/<REDACTED>/build/macosx-arm64/vcpkg_installed/arm64-osx/lib/libwolfssl.a

```